<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWMainWind</class>
 <widget class="QMainWindow" name="QWMainWind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWMainWind</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuedit">
    <property name="title">
     <string>edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menufont">
    <property name="title">
     <string>font</string>
    </property>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="separator"/>
    <addaction name="actionUnder"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionToolbarLab"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuedit"/>
   <addaction name="menufont"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnder"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCut">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/cut.bmp</normaloff>:/images/cut.bmp</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/120.bmp</normaloff>:/images/120.bmp</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/paste.bmp</normaloff>:/images/paste.bmp</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/BLD.BMP</normaloff>:/images/BLD.BMP</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/ITL.BMP</normaloff>:/images/ITL.BMP</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="toolTip">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnder">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/DBLUNDR.BMP</normaloff>:/images/DBLUNDR.BMP</iconset>
   </property>
   <property name="text">
    <string>Under</string>
   </property>
   <property name="toolTip">
    <string>Under</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/exit.bmp</normaloff>:/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/open3.bmp</normaloff>:/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/103.bmp</normaloff>:/images/103.bmp</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/20.ico</normaloff>:/images/20.ico</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="toolTip">
    <string>Font</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/new2.bmp</normaloff>:/images/new2.bmp</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionToolbarLab">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/430.bmp</normaloff>:/images/430.bmp</iconset>
   </property>
   <property name="text">
    <string>ToolbarLab</string>
   </property>
   <property name="toolTip">
    <string>ToolbarLab</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>QWMainWind</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
